### DISCUSSION TITLE
Please add Ukrainian language, file with translation attached

### DISCUSSION THREAD

**Discussion by ualenet** [drakkan/sftpgo#1698] [Ideas]
Hello!
Please add Ukrainian language.

    const lngs = {
        en: { nativeName: 'English' },
        it: { nativeName: 'Italiano' },
        ua: { nativeName: 'Українська' }
    };

[translation.json.zip](https://github.com/user-attachments/files/16256299/translation.json.zip)

**Comment by drakkan** [comment]
Thank you. I have added Ukrainian to [Crowdin](https://crowdin.com/project/sftpgo). 
Anyway we don't accept translations from causual users, please check our [contribution guidelines](https://sftpgo.github.io/latest/web-interfaces/#internationalization).
